apps/services/views.py

```
@@ -0,0 +1,92 @@
+            serializer.save(vendor=user.vendor_profile)
+            return
+
+        if not (user.is_staff or user.is_superuser):
+            raise PermissionDenied('Only vendors or staff can create services.')
+
+        if vendor is None:
+            raise PermissionDenied('Staff must specify a vendor when creating services.')
+
+        serializer.save()
+
+    def _assert_can_mutate(self, instance):
```

Ruff (0.14.0)

Missing return type annotation for private function `_assert_can_mutate`

Add return type annotation: `None`

(ANN202)